app/src/main/java/com/example/android/weatherapp/utilities/Helper.kt

```diff
@@ -0,0 +1,49 @@
+package com.example.android.weatherapp.utilities
+
+import java.text.DateFormat
+import java.text.SimpleDateFormat
+import java.util.*
+import kotlin.math.round
+
+private const val DATE_FORMAT = "MMMM dd, yyyy"
+
+object Helper {
+    const val APP_ID = "4dfdfc7144138f43bfa36b5b3a4b097f"
+
+    fun getDayFromDate(inputDate: String): String{
+        val format: DateFormat = SimpleDateFormat("yyyy-MM-dd hh:mm:ss", Locale.ENGLISH)
+        val date: Date = format.parse(inputDate)
+        val curr_date = SimpleDateFormat("EE").format(date)
+        return curr_date.toString()
+    }
+
+    fun getTimeFromDate(inputDate: String): String{
+        val format: DateFormat = SimpleDateFormat("yyyy-MM-dd hh:mm:ss", Locale.ENGLISH)
+        val date: Date = format.parse(inputDate)
+        val time = SimpleDateFormat("hh:mm").format(date)
+        return time.toString()
+    }
+
+    fun getCurrentDate(): String {
+        val simpleDateFormat = SimpleDateFormat(DATE_FORMAT)
+        val date: String = simpleDateFormat.format(Date())
+        return date
+    }
+
+    fun convertToCelsius(fahrenheit: Double): Double{
+        val converted = ((fahrenheit.toDouble() - 273.15))
+        return roundDecimalPlaces(converted)
+    }
+
+    fun roundDecimalPlaces(input: Double): Double{
```

Comment:
What does this method do?

Reply:
It rounds off the value to one decimal place.